### ISSUE TITLE
The network's configuration of CARLA_DDPG

### ISSUE THREAD

**Issue by fangchuan** [NervanaSystems/coach#275]
Hi，I try to run The CARLA_DDPG preset in coach，with The help of The document，I run it successfully. However，i want to dig into the implementation of DDPG in coach, I have reviewed The code of CARLA_DDPG.py, and i have figured out both the network of actor and critic, as the below picture, who can help me revise the understanding and give me some supplementary advise?

**Comment by fangchuan** [comment]
![ddpg_actor_critic_network](https://user-images.githubusercontent.com/8519718/55320672-40368880-54aa-11e9-9d57-c7c13ba77fc0.jpeg)

**Comment by huang-CX** [comment]
Hi, it is hard to be convergence if the network parameters are randomly initialized. I have tried to initialize the convolutional layers with pretrained parameters, and it performed better.

**Comment by galnov** [comment]
@fangchuan you can use the --print_networks_summary flag to print the networks' structure.

**Comment by fangchuan** [comment]
@huang-CX ,where did you find the pretrained parameter?

**Comment by fangchuan** [comment]
hi, I have a question when I print the network's summary: although there are batch_normalization layers inserted between each convolutional layers or dense layers in the network, it does not set the `is_training` flag to True. It means the network didn't use batch_normalization at all, why you did not use batch_normalization in DDPG? I'm looking forward to your answer, @galnov

**Comment by fangchuan** [comment]
Another question in CARLA_DDPG is about the activation_fn used in critic_network, whether the critic_network use the relu activation function in output head(v_head)?  Because when i use the relu in critic_network's output layer in my ddpg implementation, I found it was stuck in the dead area at the beginning of training.

**Comment by huang-CX** [comment]
@fangchuan  you can use a pretrained VGG to extract feature from raw image.

**Comment by galnov** [comment]
> hi, I have a question when I print the network's summary: although there are batch_normalization layers inserted between each convolutional layers or dense layers in the network, it does not set the `is_training` flag to True. It means the network didn't use batch_normalization at all, why you did not use batch_normalization in DDPG? I'm looking forward to your answer, @galnov

Batch norm fixed in [this PR](https://github.com/NervanaSystems/coach/pull/353). Note that it is still disabled in DDPG as results were better without it.

**Comment by galnov** [comment]
Closing as the questions have been answered. If this is still not resolved, please reopen.
